auto/update_adv_stats.R

```diff
@@ -29,13 +29,13 @@ scrape_advstats <- function(){
     return(FALSE)
   }
 
-  cli::cli_alert("Now scraping {nrow(game_ids)} games")
+  cli::cli_alert("Now scraping {nrow(game_ids)} game{?s}")
 
   #' Scrape Incomplete Games
   scrape_games <- game_ids %>%
-    dplyr::mutate(adv = purrr::map(pfr_game_id,
+    dplyr::mutate(adv = purrr::map(cli::cli_progress_along(pfr_game_id),
                                    purrr::possibly(
-                                     pfr_game_adv_stats,
+                                     .f = function(i) pfr_game_adv_stats(pfr_game_id[[i]]),
                                      otherwise = list(pass = NULL,
```

Comment:
@tanho63 there is a problem in this code chunk. When scraping fails and possibly returns the list of NULLs, we will see a failurte in the dplyr::filter() because the column "stats" won't exist. I am running out of time. Could you look into this?

Reply:
Right now it should be possible to reproduce the problem. There is one finished game that can't be scraped

Reply:
👍 can look this aft yep

Reply:
I think this is good now, pls check and lmk